config/routes.rb

```diff
@@ -85,6 +85,8 @@
 
   resources :offices do
     resources :business_entities
+    get '/business_entities/:id/confirm' => 'business_entities#confirm', as: 'confirm_deactivate_business_entity'
+    post '/business_entities/:id/deactivate' => 'business_entities#deactivate', as: 'deactivate_business_entity'
```

Comment:
These 2 rules are related to the resource and ideally would be nested in it: 

```ruby
resources :business_entities do
  get :confirm, on: :member
  post :deactivate, on: :member
end
```

Something between these lines.